packages/shared/abstract-sdk/src/lib/utils.ts

```diff
@@ -38,3 +40,5 @@ export const placeArgsToString = (message: string, args?: SdkMessageArgs): strin
   }
   return message
 }
+
+export const toHexString = (value: BigNumber) => `0x${value.toString(16)}`
```

Comment:
maybe let's move it under amount utils?

Reply:
I think its general not really connected to amount. Using it for example in `gasLimitToHexWithFallback`.